lib/src/base-styles/_nx-colors.scss

```diff
@@ -55,6 +55,17 @@
   --nx-color-binary-donut-chart-background-fill: var(--nx-swatch-indigo-95);
   --nx-color-binary-donut-chart-arc-stroke: var(--nx-swatch-teal-60);
 
+  // NxGlobalHeader
+  --nx-color-global-header-border: var(--nx-color-border);
+
+  //NxPageHeader variables
+  --nx-color-page-header-background: var(--nx-color-component-background);
+  --nx-color-page-header-border-bottom: var(--nx-color-border);
```

Comment:
Also a small thing, and also not 100%, what are your thoughts of just renaming it to just `--nx-color-page-header-border`.
(Since there are no other border edges, and also for consistency, there are other variables with just `-border` that is assigned to just one edge)

Reply:
> Also a small thing, and also not 100%, what are your thoughts of just renaming it to just `--nx-color-page-header-border`. (Since there are no other border edges, and also for consistency, there are other variables with just `-border` that is assigned to just one edge)

I'm fine with that, so updated here: [3d58df9](https://github.com/sonatype/sonatype-react-shared-components/pull/979/commits/3d58df9ed3a614bbb6dbd7d27a441f00c3fda659). This raises another question. Currently the underline (technically a bottom border) of the active header link is `--nx-color-page-header-active-link-border-bottom`. Should that be changed to remove the `-bottom`, or changed to `underline`? Or left as is?

Reply:
> > > Also a small thing, and also not 100%, what are your thoughts of just renaming it to just `--nx-color-page-header-border`. (Since there are no other border edges, and also for consistency, there are other variables with just `-border` that is assigned to just one edge)
> > 
> > 
> > I'm fine with that, so updated here: [3d58df9](https://github.com/sonatype/sonatype-react-shared-components/pull/979/commits/3d58df9ed3a614bbb6dbd7d27a441f00c3fda659). This raises another question. Currently the underline (technically a bottom border) of the active header link is `--nx-color-page-header-active-link-border-bottom`. Should that be changed to remove the `-bottom`, or changed to `underline`? Or left as is?
> 
> Ooo I totally missed that, I think that one should be renamed to `-bottom` as well

Hmm, I'm curious since this looks like a normal link was it intentionally set to a different value in dark mode than `--nx-color-link`? (teal-60 vs teal-65) 🤔 

I wonder if it's a good idea to reach out to the designers to confirm if this was maybe intended to be the same as the link color. 
If so we can just use `--nx-color-link` and remove this variable.

Reply:
>Ooo I totally missed that, I think that one should be renamed to -bottom as well

Sorry, it's currently named `--nx-color-page-header-active-link-border-bottom`, so leave it as is? And good idea, I am reaching out to design now to see if that colour choice was intentional

Reply:
> Ah I'm so sorry, still waking up, I meant removing the `-bottom` 😸

Removed `-bottom` and updated the colour of the link + its bottom border here: [3e4252e](https://github.com/sonatype/sonatype-react-shared-components/pull/979/commits/3e4252e39b73f3cd48b24cc18e60906a13bff3ef)

Reply:
> > Ah I'm so sorry, still waking up, I meant removing the `-bottom` 😸
> 
> Removed `-bottom` and updated the colour of the link + its bottom border here: [3e4252e](https://github.com/sonatype/sonatype-react-shared-components/pull/979/commits/3e4252e39b73f3cd48b24cc18e60906a13bff3ef)

Thanks Emily! I think we could delete `--nx-color-page-header-active-link-border` variable entirely and just use `--nx-color-link` directly for that

Reply:
> > > Ah I'm so sorry, still waking up, I meant removing the `-bottom` 😸
> > 
> > 
> > Removed `-bottom` and updated the colour of the link + its bottom border here: [3e4252e](https://github.com/sonatype/sonatype-react-shared-components/pull/979/commits/3e4252e39b73f3cd48b24cc18e60906a13bff3ef)
> 
> Thanks Emily! I think we could delete `--nx-color-page-header-active-link-border` variable entirely and just use `--nx-color-link` directly for that

I don't think so unfortunately, because the link's border is different in light mode!

Reply:
> > > Ah I'm so sorry, still waking up, I meant removing the `-bottom` 😸
> > 
> > 
> > Removed `-bottom` and updated the colour of the link + its bottom border here: [3e4252e](https://github.com/sonatype/sonatype-react-shared-components/pull/979/commits/3e4252e39b73f3cd48b24cc18e60906a13bff3ef)
> 
> Thanks Emily! I think we could delete `--nx-color-page-header-active-link-border` variable entirely and just use `--nx-color-link` directly for that

Good catch, fixed here: [226a681](https://github.com/sonatype/sonatype-react-shared-components/pull/979/commits/226a68136594d3071eec71b6a3a42a1697f091ce)